vim_template/langs/php/php.bundle

```diff
@@ -1,2 +1,3 @@
 "" PHP Bundle
 Plug 'arnaud-lb/vim-php-namespace'
+Plug 'phpactor/phpactor', {'for': 'php', 'do': 'composer install'}
```

Comment:
Is this a php related plugin meant to be in this PR?

Reply:
remove this line for this PR pls